<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_users</class>
 <widget class="QWidget" name="main_users">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Группа:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushAddGroup">
         <property name="text">
          <string>Добавить
 группу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushDelGroup">
         <property name="text">
          <string>Удалить
 группу</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="table" name="tableUsers"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushAddUser">
           <property name="text">
            <string>Создать
пользователя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushDelUser">
           <property name="text">
            <string>Удалить
пользователя</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushEdit">
           <property name="text">
            <string>Редактировать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>table</class>
   <extends>QTableWidget</extends>
   <header>Widgets/table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
